src/js/components/useToast.ts

```diff
@@ -87,21 +87,21 @@ const getToastElement = (template?: string): HTMLElement => {
 };
 
 // We need to use a template, in order to generate the toast markup on the fly
-const cloneToastTemplate = (containerToAppend: HTMLElement, toastContainer?: HTMLElement): HTMLElement | false => {
+const cloneToastTemplate = (toastContainer: HTMLElement, overrideContainer?: HTMLElement): HTMLElement | false => {
   let toastTemplate;
 
-  if (toastContainer) {
-    toastTemplate = toastContainer.querySelector<HTMLTemplateElement>(selectorsMap.toast.template);
+  if (overrideContainer) {
+    toastTemplate = overrideContainer.querySelector<HTMLTemplateElement>(selectorsMap.toast.template);
```

Comment:
I think this will cause some problems

Try to initialise a toast with the container already present in the page

Then try to override it with this

Then show both toast at the same time

Reply:
For example:

```js
const {instance: test} = window.Theme.components.useToast('test')
const {instance: overrideTest} = window.Theme.components.useToast('overrideTest', options: {<put the overriden markup>})

test.show();
overrideTest.show();
```

One of these toasts will be under the other, because the container is duplicated :p

It works if you change the overridden container position, bottom: 0 for example, but not in the case you place the container at the same place than the already present one :P

Reply:
We create two instances and then call, so we should had two toasts at stack:

![Screenshot from 2022-03-04 07-50-14](https://user-images.githubusercontent.com/85633460/156766871-f72944a4-afb5-4d65-93ea-7783ace1199e.png)

What's the problem?

Reply:
Oh ok, just understood that you didn't change anything about the logic I introduced, so it's fine

The overridden container will never be used, we should probably get rid of it as it's useless